src/app/Dashboard/AddCard.tsx

```diff
@@ -0,0 +1,145 @@
+ * person obtaining a copy of this software, associated documentation and/or data
+ * (collectively the "Software"), free of charge and under any and all copyright
+ * rights in the Software, and any and all patent rights owned or freely
+ * licensable by each licensor hereunder covering either (i) the unmodified
+ * Software as contributed to or provided by such licensor, or (ii) the Larger
+ * Works (as defined below), to deal in both
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import {
+  Bullseye,
+  Button,
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Select,
+  SelectOption,
+  Stack,
+  StackItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+import { Wizard, WizardStep } from '@patternfly/react-core/dist/js/next';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { useDispatch } from 'react-redux';
+import { StateDispatch } from '@app/Shared/Redux/ReduxStore';
+import { addCardIntent } from '@app/Shared/Redux/DashboardConfigActions';
+import { DashboardCards, getConfigByName, getConfigByTitle } from './Dashboard';
+
+interface AddCardProps {}
+
+export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardProps) => {
+  const [showWizard, setShowWizard] = React.useState(false);
+  const [selection, setSelection] = React.useState(DashboardCards[0].title);
+  const [selectOpen, setSelectOpen] = React.useState(false);
+  const dispatch = useDispatch<StateDispatch>();
+
+  const options = React.useMemo(() => {
+    return [
+      ...DashboardCards.map((choice, idx) => (
+        <SelectOption key={idx} value={choice.component.name}>
+          {choice.title}
```

Comment:
Not sure what happened but seems to run into error if I select an option for card type.

Reply:
I think the value should be `choice.title`?

Reply:
Hmm, can't seem to reproduce this. Maybe try clearing site data for `localhost` so the localstorage is reset?

Reply:
Yehh still happens if I clear localStorage

```
Uncaught Error: Unknown card type selection: AutomatedAnalysisCard
getConfigByTitle Dashboard.tsx:91
AddCard AddCard.tsx:120
React 19
```

Reply:
I am on the last commit on upstream now at [b0ea246](https://github.com/cryostatio/cryostat-web/pull/731/commits/b0ea2466c4b85bff3f564784f2f8fe4147548dfe)

Reply:
Oh wait, I triggered it now. I just didn't quite understand what you meant.